pygeoapi/process/hello_world.py

```diff
@@ -118,7 +118,7 @@ def __init__(self, processor_def):
 
         super().__init__(processor_def, PROCESS_METADATA)
 
-    def execute(self, data):
+    def execute(self, data, outputs = None):
```

Comment:
As an extension to this, we could also already implement the behavior for `outputs` here, maybe something along the lines of this:
```python
produced_outputs = {}
if outputs is None or 'echo' in outputs:
    produced_outputs['echo'] = value
return mimetype, produced_outputs
```